<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loanBookWindow</class>
 <widget class="QDialog" name="loanBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pegar livro Emprestado</string>
   </property>
  </widget>
  <widget class="QLabel" name="bookLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loanButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Pegar emprestado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>anterior</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>proximo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
